recon_surf/N4_bias_correct.py

```diff
@@ -15,18 +15,16 @@
 
 
 # IMPORTS
+import image_io as iio
 import argparse
 import sys
-from pathlib import Path
-from typing import Optional, cast, Tuple
 import logging
-
 import SimpleITK as sitk
 import numpy as np
-from numpy import typing as npt
-
-import image_io as iio
 
+from pathlib import Path
```

Comment:
overall the ordering of imports is incorrect, the groups we had before were good, please only reorder within the groups.

group 1: python native modules
group 2: external modules, eg. torch, numpy
group 3: internal modules, e.g. here image_io

Reply:
should be propagated to changes to all files (of this PR)

Reply:
> group 1: python native modules
> group 2: external modules, eg. torch, numpy
> group 3: internal modules, e.g. here image_io

Regrouped as follows: 
### Group 1: Python native modules
import argparse
import logging
import sys
from pathlib import Path
from typing import Optional, cast, Tuple

### Group 2: External modules
import SimpleITK as sitk
import numpy as np
from numpy import typing as npt

### Group 3: Internal modules
import image_io as iio